core/src/main/java/org/elasticsearch/common/xcontent/XContentBuilder.java

```diff
@@ -944,6 +944,10 @@ public XContentBuilder dateValueField(XContentBuilderString rawFieldName, XConte
         return this;
     }
 
+    public XContentBuilder timeValueField(String rawFieldName, String readableFieldName, TimeValue timeValue) throws IOException {
```

Comment:
I'll have a look. This might be another one of those "we should fix it it master" things. Or not.